### PR TITLE
MEN-5186 Monitor: full integration test on config push

### PR DESCRIPTION
**Pull request by merlin-northern**
They go together:
* https://github.com/mendersoftware/integration/pull/1742
* https://github.com/mendersoftware/monitor-client/pull/95
* https://github.com/mendersoftware/devicemonitor/pull/66

ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
the failure in open source integration is unrelated.

File: tests/tests/test_monitor_client.py [hunk 1]
```diff
@@ -1439,3 +1439,54 @@ def test_monitorclient_send_saved_alerts_on_network_issues(
...
+        )
+        time.sleep(2 * wait_for_alert_interval_s)
+        device_monitor = DeviceMonitor(auth)
+        configuration = device_monitor.get_configuration(devid)
+        assert len(configuration) == 4
+        for entity in [
+            {"name": "crond.sh", "type": "service"},
+            {"name": "dbus.sh", "type": "service"},
+            {"name": "syslog.sh", "type": "log"},
+            {"name": "clientlogs.sh", "type": "log"},
```
Comment (alfrunes) [inline tests/tests/test_monitor_client.py:1481]: Are there any example use cases where the device would publish a complete descriptor at this moment?
For example:
```json
{"name": "clientlogs.sh", "type": "log", "status": "OK?", "log": {"pattern": "any err.*or[s]?", "path": "@journalctl -u trouble", "expire_seconds": 42069}}
```

File: tests/tests/test_monitor_client.py [hunk 2]
```diff
@@ -1439,3 +1439,54 @@ def test_monitorclient_send_saved_alerts_on_network_issues(
...
+        prepare_log_monitoring(
+            mender_device,
+            "clientlogs",
+            "@journalctl -u mender-client -f",
+            "[Ee]rror.*",
+            use_ctl=True,
+        )
+        time.sleep(2 * wait_for_alert_interval_s)
+        device_monitor = DeviceMonitor(auth)
+        configuration = device_monitor.get_configuration(devid)
```
Comment (alfrunes) [inline tests/tests/test_monitor_client.py:1475]: A poll loop over a longer time-span is more stable than a static wait. You never know if there's going to be any hiccups in the meantime.

**Review by lluiscampos — APPROVED** [review]

**Comment by merlin-northern** [comment]
last push is just a rebase on top of master

**Comment by merlin-northern** [comment]
last pipeline: https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/412090483

**Pull request merged**